File: send_email.py
import yagmail
import os
import datetime

password = os.environ.get('EMAIL_PASSWORD')
sender = 'fvillatoro99@gmail.com'
recipient = 'fvillatoro99@gmail.com'

today = datetime.date.today().strftime('%B %d, %Y')

subject = f'DAILY COLD PLUNGE REMINDER FOR {today}'
body = """

test

"""

try:
    yag = yagmail.SMTP(sender, password)
    yag.send(to=recipient, subject=subject, contents=body)
    print("Email sent success")

except Exception as e:
    print("Error sending email: {e}")

File: send_email.js
//
// send_email.js
// This script sends an email reminder using Nodemailer.
//

// Use the 'nodemailer' library to send emails
const nodemailer = require('nodemailer');

// Set up the transporter using SMTP with an App Password from GitHub Secrets
const transporter = nodemailer.createTransport({
  service: 'gmail',
  auth: {
    user: process.env.EMAIL_USER, // Your Gmail address
    pass: process.env.EMAIL_PASSWORD // Your App Password from GitHub Secrets
  }
});

// Define the email options
const mailOptions = {
  from: process.env.EMAIL_USER,
  to: process.env.EMAIL_USER,
  subject: 'Daily Cold Plunge Reminder',
  text: `
Hey there,

DID YOU DO YOUR COLD PLUNGE?!?!?!?

Love,
yourself
  `
};

// Send the email
transporter.sendMail(mailOptions, function(error, info){
  if (error) {
    console.log("An error occurred!");
    console.log(error);
  } else {
    console.log('Email sent: ' + info.response);
  }
});

